doc/pymode.txt

```diff
@@ -131,6 +131,18 @@ Setup pymode |quickfix| window.
     let g:pymode_quickfix_minheight = 3
     let g:pymode_quickfix_maxheight = 6
 
+Set pymode |preview| window height.                     *'g:pymode_preview_height'*
+Preview window is used to show documentation and ouput from |pymode-run|.
+>
+    let g:pymode_preview_height = &previewheight
```

Comment:
@fpob which one is the default value for `previewheight`?

Reply:
Default is 12 [[`:help 'previewheight'`](https://vimhelp.appspot.com/options.txt.html#%27previewheight%27)].